### ISSUE TITLE
Allow comment message to show `npx` over `npm add`

### ISSUE THREAD

**Issue by manuel3108** [stackblitz-labs/pkg.pr.new#263]
This comment always looks something like this:

```
npm add https://pkg.pr.new/repo/package@1
```

We are building a CLI, and would love to have an option to make the comment display 

```
npx https://pkg.pr.new/repo/package@1
```

**Comment by manuel3108** [comment]
Mhmm, that looks like a lot of boilerplate code that we need to copy over and maintain. Would it be possible that you publish some helper methods as part of `pkg-pr-new` that would dramatically simplify working the api?

Most of the code you linked in the example would probably be the same for 85% of the users, so I was wondering if it would be useful to have a common abstraction? I think the `output.json` that you suggested seems like the right way to consume the results.

**Comment by Aslemammad** [comment]
you are right! let's see if more folks would need this and i'd be happy to implement it!
